x/liquidstaking/keeper/liquidstaking.go

```diff
@@ -1443,6 +1344,15 @@ func (k Keeper) IsSufficientInsurance(ctx sdk.Context, insurance types.Insurance
 	return insBal.Amount.GTE(minimumCollateral.Amount)
 }
 
+// IsEnoughToCoverSlash checks whether insurance has sufficient balance to cover slashing or not.
+func (k Keeper) IsEnoughToCoverSlash(ctx sdk.Context, insurance types.Insurance) bool {
+	params := k.slashingKeeper.GetParams(ctx)
+	downTimePenaltyAmt := params.SlashFractionDowntime.MulInt(sdk.NewInt(10)).Ceil().TruncateInt()
```

Comment:
**`params.SlashFractionDowntime` is Fraction, It must be `Mul` with ChunkSize**

Reply:
Fixed it in [this commit](https://github.com/b-harvest/Canto/pull/9/commits/57fee4ba046390d1028e48117c9327e40a49eba4)